tests/test_tasks.py

```diff
@@ -24,6 +24,7 @@
 from unittest import mock
 
 import pytest
+from test_main import generate_noxfile_options  # noqa: F401
```

Comment:
This is a fixture so you don't need to (and shouldn't) import it, I think

Reply:
Thanks @edgarrmondragon, seems I'd get a `fixture 'generate_noxfile_options' not found` error if I do that. I could have moved it to a `conftest.py` but since there wasn't one already I decided for this approach instead. Thoughts?

Reply:
Ah, yeah it's defined in a different module. It's probably fine to add a `conftest.py` with this fixture alone imo.

Reply:
@edgarrmondragon Moved to `conftest.py` in a64e632fba1bf497ddf4662b458e22763cbcaecc